src/lazyload-image.directive.ts

```diff
@@ -18,13 +18,13 @@ import {
 import { getScrollListener } from './scroll-listener';
 import { lazyLoadImage } from './lazyload-image';
 
-const target = typeof window !== 'undefined' ? window : undefined;
+const windowTarget = typeof window !== 'undefined' ? window : undefined;
 
 interface LazyLoadImageDirectiveProps {
     lazyImage: string;
     defaultImage: string;
     errorImage: string;
-    scrollTarget: Object;
+    scrollTarget: HTMLElement;
```

Comment:
Since we add a type here we may also add it in `scroll-listener.ts`. You do not have to do anything here, I created a separated issue: #270